src/snowflake/snowpark/modin/plugin/_internal/frame.py

```diff
@@ -271,10 +305,18 @@ def data_column_snowflake_quoted_identifiers(self) -> list[str]:
         ]
 
     def quoted_identifier_to_snowflake_type(self) -> dict[str, DataType]:
```

Comment:
we probably should rename it to `quoted_identifier_to_type` since it may contain snowpark pandas type now.